search.html

```diff
@@ -286,6 +287,102 @@ <h2><a class="anchor" id="keyvalue"></a>Key-Value Pair Annotation Search</h2>
 
+<h2><a class="anchor" id="advanced"></a>Advanced Search</h2>
+
+<p>To understand basics of Search in OMERO, or to search for Key-Value pairs, please read also the <a href="http://help.openmicroscopy.org/search.html#tips"
+    target="_blank" class="external-url">Tips (wildcars),</a> <a href="http://help.openmicroscopy.org/search.html#keyword"
+    target="_blank" class="external-url">Keyword searching</a> and <a href="http://help.openmicroscopy.org/search.html#keyvalue"
+    target="_blank" class="external-url">Key-Value Pair Annotation Search</a> chapters. There is no difference in searching for Key-Value pairs between the basic and the <span class="bold">Advanced</span> search tabs.</p>
+
+<p>Advanced search is available in OMERO.web only.</p>
+
+<p>Inside the search page, click on the <span class="bold">Advanced</span> tab in the left-hand pane.</p>
+
+<p>OMERO.server uses <a href="https://lucene.apache.org/">Lucene</a> to perform the search commands. The <span class="bold">Advanced</span> tab accepts directly the Lucene queries, with no parsing. Follow the steps below to get started with Lucene queries.
+
+<p>Construct searches for a string in any <span class="bold">Field</span> as listed in <a href="https://docs.openmicroscopy.org/latest/omero/developers/Modules/Search.html#field-names"
+    target="_blank" class="external-url">the table of fields in the documentation</a> using the syntax: <span class="code-example">Field:string</span>.</p>
+
+<table>
+ <tr>
+  <th>Field</th>
+  <th>Comment</th>
+ </tr>
+ <tr>
+  <td>name</td>
+  <td>name of the field</td>
+ </tr>
+ <tr>
+  <td>details.owner.omeName</td>
+  <td>login name of the owner</td>
+ </tr>
+ <tr>
+  <td>details.owner.firstName</td>
+  <td>first name of the owner</td>
+ </tr>
+ <tr>
+  <td>details.owner.lastName</td>
+  <td>last name of the owner</td>
+ </tr>
+ <tr>
+  <td>details.group.name</td>
+  <td>group name of the group</td>
+ </tr>
+ <tr>
+  <td>tag</td>
+  <td>A Tag attached to the object</td>
+ </tr>
+ <tr>
+  <td>annotation</td>
+  <td>Tag, Key-Value Pairs, Comment, but not Attachment.</td>
+ </tr>
+ <tr>
+  <td>annotation.ns</td>
+  <td>A namespace of an annotation (Tag, Key-Value Pairs, Comment and Attachment)</td>
+ </tr>
+ <tr>
+  <td>annotation.type</td>
+  <td>Type of an annotation, in the syntax TagAnnotation for Tags, MapAnnotation for Key-Value Pairs, CommentAnnotation for Comment, FileAnnotation for Attachment and LongAnnotation for Ratings.</td>
+ </tr>
+ <tr>
+  <td>file.name</td>
+  <td>Name of original file attached to an object. Note: When a file is attached to an object in OMERO, OMERO.server creates an object called OriginalFile. It is actually the name of this OriginalFile which is searched for. The OriginalFile name and the name of the uploaded file match unless the OriginalFile was renamed by an explicit action of the owner or admin.</td>
```

Comment:
Here and below, I wouldn't worry about whether the original name has been preserved. If the user is using workflows where the original name is not being preserved, they will likely be aware of it already. If there's any discrepancy it's not really a search issue.